charts/oes/config/oes-gate/gate.yml

```diff
@@ -1,3 +1,6 @@
+ok-http-client:
+  interceptor:
+    skipHeaderCheck: true
```

Comment:
@girichinna27 by default are we disabling or we will have the flag.

Reply:
By default - it will be true only @saitejaopsmx